themes/assets/assets.less

```diff
@@ -10,6 +10,9 @@
 @monsterBorderImage       : url('/assets/monsterBorderFancy.png');
 @codeBorderImage          : url('/assets/codeBorder.png');
 @classTableDecoration     : url('/assets/classTableDecoration.png');
+@coverPageLogo            : url('/assets/coverPageLogo.png');
+@coverPageBanner          : url('/assets/coverPageBanner.png');
+@coverPageHorizontalRule  : url('/assets/coverPageHorizontalRule.png');
```

Comment:
I think this should be edited to remove the existing drop shadow.  Since the other two graphic elements have no drop-shadow baked into them, this one should match that expectation.  Then, as we do with the others, apply the drop-shadow with css.

(or the other way around-- bake the drop-shadow into these files and remove the css).

Reply:
Agreed. Let's keep the graphics clean where possible and apply effects via css

Reply:
so, removing the drop shadow from the horizontal rule? that'd be no problem

Reply:
@calculuschild @Gazook89 should i make them vectorial? that would make them use less space, right?

Reply:
Yes, if they appear the same and don't have any weird behavior with positioning and sizing compared to pngs.

Reply:
they appear fine and behave as a png would.